returnn/datasets/lm.py

```diff
@@ -125,14 +125,16 @@ def __init__(self,
       orth_symbols = open(orth_symbols_file).read().splitlines()
       self.orth_symbols_map = {sym: i for (i, sym) in enumerate(orth_symbols)}
       self.orth_symbols = orth_symbols
-      self.labels["data"] = orth_symbols
+      reverse_map = {i: sym for (i, sym) in enumerate(orth_symbols)}
+      self.labels["data"] = [sym for (i, sym) in sorted(reverse_map)]
```

Comment:
I don't understand this. Isn't this just the same as before?

Reply:
Ah yes, here we have no explicit indices anyway.